common/test/acceptance/pages/studio/overview.py

```diff
@@ -268,15 +268,15 @@ def subsection_expanded():
 
         # Need to click slightly off-center in order for the click to be recognized.
         ele = self.browser.find_element_by_css_selector(self._bounded_selector('.ui-toggle-expansion .fa'))
-        ActionChains(self.browser).move_to_element_with_offset(ele, 4, 4).click().perform()
+        ActionChains(self.browser).move_to_element_with_offset(ele, 8, 8).click().perform()
         self.wait_for_element_presence(self._bounded_selector(self.ADD_BUTTON_SELECTOR), 'Subsection is expanded')
 
         EmptyPromise(
             lambda: subsection_expanded() != currently_expanded,
             "Check that the container {} has been toggled".format(self.locator)
         ).fulfill()
 
-        self.browser.execute_script("jQuery.fx.off = false;")
+        enable_animations(self)
```

Comment:
I think you should reenable animations in a finally clause so that if there is an exception it doesn't leave animations disabled for all the other tests.